mix.exs

```diff
@@ -0,0 +1,48 @@
+defmodule EredisCluster.Mixfile do
+  use Mix.Project
+
+  @source_url "https://github.com/Nordix/eredis_cluster/"
+  @version String.trim(File.read!("VERSION"))
+
+  def project do
+    [
+      app: :eredis_cluster,
+      deps: deps(),
+      description: "An erlang wrapper for eredis library to support cluster mode",
+      docs: [
+        main: "readme",
+        extras: ["README.md", "doc/eredis_cluster.md", "doc/eredis_cluster_monitor.md"],
```

Comment:
Isn't `eredis_cluster_monitor` internal?

Reply:
We have theses (see below) that was requested by E. at start, but maybe these should be moved, or handled differently..
`get_cluster_slots` is at least used internally aswell, but `get_cluster_nodes` is he odd animal only provided as an api.
But we skip doc exporting for them anyway I guess

  - `eredis_cluster_monitor:get_cluster_nodes/0`: Get cluster nodes information list (CLUSTER NODES)
  - `eredis_cluster_monitor:get_cluster_slots/0`: Get cluster slots information (CLUSTER SLOTS)

Reply:
I've thought I had added wrappers for these in the `eredis_cluster` module, but it seems I only added wrappers for get_pool_by_command/1, get_pool_by_key/1, get_all_pools/0.

Should we add wrappers for get_cluster_{nodes,slots}/0 as well?

Reply:
As discussed in parallel my understanding was that we keep the `doc/eredis_cluster_monitor.md` here since everything except `get_cluster_nodes/0` and `get_cluster_slots/0` are marked as private, and we will possibly add wrappers for them.